RodiSales87/pta-squad-drigo
# Feat/profile

### What I did

- Implemented the Profile page with functionality for filtering and displaying game sessions (partidas) based on open, closed, or historical status.
- Added date range filter to allow searching by start and end dates.
- Integrated dynamic data rendering with React's state management.

### How to test

1. Open the terminal and go to the folder of this project.
2. In the terminal:

`git checkout develop`
`git pull`
`git checkout feat/profile`
`git pull origin feat/profile`
`npm install`
`npm run dev`

3. In your browser, go to http://localhost:3000/.
4. Ensure the profile page works correctly, with the username input field, search button, and dynamic display of game sessions (open, closed, historical).
5. Test the date range filter by selecting different date ranges and checking if the filtered sessions display as expected.
